index.html

```diff
@@ -4,311 +4,313 @@
-                    <a class="sub-link" href="#">Щербеты</a>
-                  </li>
-                  <li class="sub-item">
-                    <a class="sub-link" href="#">Фруктовый лед</a>
-                  </li>
-                  <li class="sub-item">
-                    <a class="sub-link" href="#">Мелорин</a>
-                  </li>
-                </ul>
+    <header class="page-header">
+      <a class="logo">
+        <img src="https://via.placeholder.com/137x56" alt="Логотип магазина Gllacy" width="137" height="56">
+      </a>
+      <nav class="main-nav">
+        <ul class="nav-site">
+          <li class="site-item">
+            <a class="site-link" href="catalog.html">Каталог</a>
+            <ul class="sub-menu">
+              <li class="sub-item">
+                <a class="sub-link" href="#">Новинки</a>
               </li>
-              <li class="site-item">
-                <a class="site-link" href="#">Доставка и оплата</a>
+              <li class="sub-item">
+                <a class="sub-link" href="#">Сливочное</a>
               </li>
-              <li class="site-item">
-                <a class="site-link" href="#">О компании</a>
+              <li class="sub-item">
+                <a class="sub-link" href="#">Щербеты</a>
               </li>
-          </ul>
-          <ul class="nav-user">
-            <li class="user-item">
-              <a class="user-phone" href="tel:+78005558628">+7 800 555-86-28</a>
-            </li>
-            <li class="user-item">
-              <a class="user-search" href="#"><span class="visually-hidden">Поиск</span></a>
-            </li>
-            <li class="user-item">
-              <a class="user-login" href="#">Вход</a>
-            </li>
-            <li class="user-item">
-              <a class="user-cart" href="#">Пусто</a>
-            </li>
-          </ul>
-        </nav>
-      </div>
-    </header>
+              <li class="sub-item">
+                <a class="sub-link" href="#">Фруктовый лед</a>
+              </li>
+              <li class="sub-item">
+                <a class="sub-link" href="#">Мелорин</a>
+              </li>
+            </ul>
+          </li>
+          <li class="site-item">
+            <a class="site-link" href="#">Доставка и оплата</a>
+          </li>
+          <li class="site-item">
+            <a class="site-link" href="#">О компании</a>
+          </li>
+        </ul>
+
+        <a class="user-phone" href="tel:+78005558628">+7 800 555-86-28</a>
 
-    <main>
+        <ul class="nav-user">
+          <li class="user-item">
+            <a class="user-search" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+          <li class="user-item">
+            <a class="user-login" href="#">Вход</a>
+          </li>
+          <li class="user-item">
+            <a class="user-cart" href="#">
+              <span class="visually-hidden">Корзина </span>
```

Comment:
А если написать так, то _(наверное)_ интонация у скринридера будет правильнее
```suggestion
              <span class="visually-hidden">Корзина: </span>
```

Reply:
согласен